src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+/* Big search styles */
+.search-panel__big {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.search-bar__big {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+  padding-left: 62px;
+  background: url("./images/Search.svg") 26px 19px no-repeat;
+  background-position: 26px center;
+  transition: all 0.5s;
```

Comment:
better to avoid "all". Replace with the style name that will smoothly change styles

Your inputs have the same styles. You can add one class (like search-bar)